src/components/Toolbar/Toolbar.jsx

```diff
@@ -43,7 +43,7 @@ function Toolbar() {
             <Link to='/'>
               <img src={homeLogo} alt='oneleif logo' />
             </Link>
-            <FaBars className='toggle' onClick={handleToggle} />
+            <FaBars size={32} className='toggle' onClick={handleToggle} />
```

Comment:
32 looks pretty big in the mobile dev tools. 24 looks much better, can you change it to that? Then should be ready to merge

Reply:
Done. Changes + commit again + push